test/modularize_test.js

```diff
@@ -84,6 +97,20 @@ var npmModularizeFiles = {
     '_maxpoolsize', '_objectpool', '_objecttypes', '_reescapedhtml', '_reinterpolate',
     '_releasearray', '_releaseobject', '_reunescapedhtml', '_setbinddata',
     '_shimisplainobject', '_shimkeys', '_slice', '_unescapehtmlchar'
+  ],
+  '3.0.0': [
+    '_arraycopy', '_cacheindexof', '_arrayeach', '_charsleftindex', '_arrayevery',
```

Comment:
ugh, already a pain retrieving these :( maybe I can bring them into node and sort